### DISCUSSION TITLE
Missing AbortSignal.timeout inside [node]

### DISCUSSION THREAD

**Discussion by vikasg603** [DefinitelyTyped/DefinitelyTyped#58837] [Issues with a @types package]
Recently NodeJS introduces AbortSignal.timeout as a global function for canceling the promise after some specified timeout.
This feature recently got introduced in Node v17.3.0 and also back ported to v16.14.0.
But in typescript suggestions, this throws an error.

Can we resolve it as soon as possible?

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
@SimonSchick we are using interface merging, so it cannot be done before upstream TS lib got updated to support this (hence your existing comment on 'abort' I assume)
